openslides/agenda/static/templates/agenda/item-detail.html

```diff
@@ -31,26 +31,53 @@ <h3 os-perm="agenda.can_manage" translate>Comment</h3>
 </div>
 
 
-<h3 translate>List of speakers</h3>
+<h3 translate>List of speakers
```

Comment:
The closing \<h3> tag is not optional: http://www.w3.org/TR/html5/syntax.html#optional-tags